doc-surrealdb_versioned_docs/version-latest/surrealql/functions/database/time.mdx

```diff
@@ -8,7 +8,20 @@ import Since from '@site/src/components/Since'
 
 # Time Functions
 
-These functions can be used when working with and manipulating datetime values.
+These functions can be used when working with and manipulating [datetime](/docs/surrealdb/surrealql/datamodel/datetimes) values.
```

Comment:
The above path will return a 404 as no such path exist, I believe the path you want to add is `[datetime](doc-surrealdb_versioned_docs/version-latest/surrealql/datamodel/datetimes.mdx)` .